mods/ra/rules/ai.yaml

```diff
@@ -190,6 +197,9 @@ Player:
 			4tnk: 15%
 			ttnk: 15%
 			stnk: 10%
+			ctnk: 2.5%
+			mrj: 0.25%
+			mgg: 0.1%
```

Comment:
The AI will build anything as long as the percentage is less than the specified value.  0.1% is going to be no different to 1% 5% or even 100% thanks to the UnitLimits you specify above.